### ISSUE TITLE
3D projection in one subplot?

### ISSUE THREAD

**Issue by florian-wagner** [lukelbd/proplot#249]
Hi @lukelbd,

first of all, thank you so much for developing `proplot`! It's a wonderful tool that would have saved me a lot of fiddling and boilerplate code in previous publications.

Quick question: Do you see any possibility to have one axis/subplot with matplotlib's 3D projection? I tried the following without success:

``` python
import matplotlib.pyplot as plt
import proplot as plot
import numpy as np

fig, axs = plot.subplots(nrows=1, ncols=4, width="20cm", sharex=3, sharey=3)

for a in axs[1:]:
    a.imshow(np.random.randn(5,5))

ax = fig.add_subplot(141, projection='3d')
```
![image](https://user-images.githubusercontent.com/2799008/114066242-cbb63180-989b-11eb-9ac9-d91414b3b1f6.png)

Thanks and cheers,
Florian

**Comment by lukelbd** [comment]
Decided to do this since the fix was relatively simple: 6df344b6 Now you can request a "3D" axes inside a proplot figure using the existing `proj` keyword.

Example: A single 3D axes

```python
fig.subplots(proj='3d')
```

Example: 3 columns, the first one is a 3D axes

```python
fig.subplots(ncols=3, proj={1: '3d'})
```

There will probably be other compatibility issues, but I think it's fair to at least add a baseline of support for 3D axes

**Comment by florian-wagner** [comment]
Thanks a lot! Much appreciated!
